src/platform/linux/kmsgrab.cpp

```diff
@@ -583,6 +583,7 @@ class display_t : public platf::display_t {
 
     for(int y = 0; y < 4; ++y) {
       if(!fb->handles[y]) {
+        sd->fds[y] = -1;
         // It's not clear wheter there could still be valid handles left.
```

Comment:
I know this wasn't your change, but since you're changing this anyway, can you fix `whether`?